101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Media Services Account."
+      }
+    },
+    "storageName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Storage."
+      }
+    },
+    "storageType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_RAGRS"
+      ],
+      "metadata": {
+        "description": "Type of the Storage. Please note you can only choose Locally Redundant, Geo-Redundant and Read-only Access Geo-Redundant as storage type to work with Azure Media Services."
+      }
+    }
+  },
+  "resources": [
+    {
+      "dependsOn": [
```

Comment:
Here is the official recommendation: https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-authoring-templates#resources